pkg/metrics/metrics_test.go

```diff
@@ -42,25 +38,26 @@ func TestRegisterCustomMetrics(t *testing.T) {
 	// then
 	// verify all metrics were registered successfully
 	for _, m := range allCounters {
-		assert.True(t, k8smetrics.Registry.Unregister(m.prometheusCounter))
+		assert.True(t, k8smetrics.Registry.Unregister(m))
 	}
 
 	for _, m := range allGauges {
-		assert.True(t, k8smetrics.Registry.Unregister(m.prometheusGauge))
+		assert.True(t, k8smetrics.Registry.Unregister(m))
 	}
+
+	// for _, m := range allGaugeVecs {
+	// 	assert.True(t, k8smetrics.Registry.Unregister(m))
+	// }
```

Comment:
Zombie?

Reply:
oh thanks, I missed that! Removed in 263f9c5 ;)